interface-impl-amf/src/main/java/org/mule/amf/impl/model/AMFImpl.java

```diff
@@ -171,7 +172,8 @@ public ApiType getType() {
   }
 
   private String renderApi() {
-    return renderApi(consoleModel.get());
+    Document document = consoleModel.get();
+    return getRenderClient().render(document);
```

Comment:
Why sometimes you specify the mime and sometimes you don't?

Reply:
It returns the API spec as it is when no mime type is specified

Reply:
Thanks for the explanation!